UnityProject/Assets/Scripts/Shuttles/CargoShuttle.cs

```diff
@@ -12,6 +12,9 @@ public class CargoShuttle : MonoBehaviour
 	[SerializeField]
 	private int dockOffset = 23;
 	private Vector3 destination;
+	private List<Vector3Int> availableSpawnSlots = new List<Vector3Int>();
+	private int shuttleWidth = 2;
```

Comment:
should have probably commented that it's `(cargoZoneWidth - 1) / 2` as it took me some time to figure out their meaning